app/controllers/pages/routes_controller.rb

```diff
@@ -40,7 +40,9 @@ def error_construct(route:, validation_error:)
     when "answer_value_doesnt_exist"
       OpenStruct.new(link: "#check-#{route.id}", message: I18n.t("page_route_card.errors.answer_value_doesnt_exist"))
     when "cannot_route_to_next_page"
-      if route.secondary_skip?
+      if !page.has_next_page?
+        OpenStruct.new(link: "#routing-question", message: I18n.t("page_route_card.errors.cannot_route_to_next_page_no_next_page"))
+      elsif route.secondary_skip?
```

Comment:
These changes don't seem to match up with the commit message

Reply:
Yes, this was left in by mistake.